src/test/java/org/mapstruct/intellij/inspection/UnmappedTargetPropertiesInspectionReportPolicyStaticImportTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.intellij.inspection;
+
+import com.intellij.codeInsight.intention.IntentionAction;
+import com.intellij.pom.java.LanguageLevel;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests if unmappedTargetPolicy could be read, if static import is used.
+ * @author hduelme
+ */
+public class UnmappedTargetPropertiesInspectionReportPolicyStaticImportTest extends BaseInspectionTest {
+
+    @Override
+    protected LanguageLevel getLanguageLevel() {
+        return LanguageLevel.JDK_1_7;
```

Comment:
We don't need to override this. Let's use the default for our tests (Java 1.8)

Reply:
Removed